src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+  isLoading: true,
+};
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      //   eslint-disable-next-line no-param-reassign
+      state.books = state.books.filter((book) => book.id !== bookId);
```

Comment:
- [ ] Let's try to not disable Linter checks in our codebase. The Linter checks are there to help you write better, error-free, and professional code. Disabling them defies its purpose. Kindly fix the Linter error 👍🏽

Reply:
@AmaduKamara Thanks for the review 💮 . Kindly suggest what should I write here or how I can change this line of code.